Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.messages.in

```diff
@@ -24,6 +24,7 @@
 // This file is generated by generate-gpup-webgl. Do not edit.
 #if ENABLE(GPU_PROCESS) && ENABLE(WEBGL)
 
+[EnabledBy=WebGLEnabled,UseGPUProcessForWebGLEnabled]
```

Comment:
There's a comment at the top of this file that says:
```
// This file is generated by generate-gpup-webgl. Do not edit.
```

This change needs to be made at https://github.com/WebKit/WebKit/blob/f6d5526e6123330465ed88dc52d6325f09e62c1a/Tools/Scripts/generate-gpup-webgl#L89

Reply:
Oh, right, fixing...

Reply:
Fixed.